homeassistant/components/asuswrt/const.py

```diff
@@ -20,12 +22,32 @@
 MODE_AP = "ap"
 MODE_ROUTER = "router"
 
+PROTOCOL_HTTP = "http"
+PROTOCOL_HTTPS = "https"
 PROTOCOL_SSH = "ssh"
 PROTOCOL_TELNET = "telnet"
 
 # Sensors
 SENSORS_BYTES = ["sensor_rx_bytes", "sensor_tx_bytes"]
 SENSORS_CONNECTED_DEVICE = ["sensor_connected_device"]
+SENSORS_CPU = [
```

Comment:
These seem like something that should be part of `sensors` and not `const`.

Reply:
This are used both by `sensors` and `bridge`, is the list of the `key` provided by api to expose sensors. Removed new sensors from this PR.